lib/client.ts

```diff
@@ -112,8 +112,8 @@ class ServiceClientStrictOptions {
 
     this.defaultRequestOptions = Object.assign({
       protocol: 'https:',
-      pathname: '/',
-      timeout: 2000
+      port: 443,
```

Comment:
This will be a breaking change for `{ protocol: 'http:' }` without an explicit port.